cmd/root.go

```diff
@@ -110,6 +110,12 @@ func init() {
 		klog.Exitf("Failed to bind show-errored-containers flag: %v", err)
 	}
 
+	rootCmd.PersistentFlags().Bool("show-old", false, "Only show charts that are not on the latest version")
+	err = viper.BindPFlag("show-old", rootCmd.PersistentFlags().Lookup("show-old"))
+	if err != nil {
+		klog.Fatalf("Failed to bind show-old flag: %v", err)
```

Comment:
Fatal has started spewing stack traces, so we're moving to Exitf everywhere. 

```suggestion
		klog.Exitf("Failed to bind show-old flag: %v", err)
```